packages/turf-polygonize/test.js

```diff
@@ -0,0 +1,51 @@
+const test = require('tape'),
+  { featureCollection, lineString, polygon } = require('@turf/helpers'),
+  polygonize = require('./'),
+  fs = require('fs'),
+  path = require('path');
+
+const directories = {
+  in: path.join(__dirname, 'test', 'in') + path.sep,
+  out: path.join(__dirname, 'test', 'out') + path.sep,
+};
+
+function getFullPath(filename, type='in') {
+  return path.join(directories[type], filename);
+}
+
+function readJsonFile(filename, type='in') {
+  try {
+    return JSON.parse(fs.readFileSync(getFullPath(filename, type)));
+  } catch(e) {
+    if (e.code !== 'ENOENT')
+      throw e;
+    return undefined;
+  }
+}
+
+function writeJsonFile(filename, data, type='out') {
+  fs.writeFileSync(getFullPath(filename, type), JSON.stringify(data));
+}
+
+fs.readdirSync(directories.in)
+  .filter(filename => !filename.startsWith('.'))
```

Comment:
I work with vim, vim generates swap files in the file's folder (`.*.swp` files). When editing tests you often forget to close the input file in order to eliminate the swap file.